core/ui.py

```diff
@@ -199,14 +199,15 @@ def GetLineSourceString(arena, line, quote_filename=False):
   return s
 
 
-def _PrintWithSpanId(prefix, msg, span_id, arena, show_code):
-  # type: (str, str, int, Arena, bool) -> None
+def _PrintWithSpanId(prefix, msg, blame_loc, arena, show_code):
```

Comment:
Can you rename this `_PrintWithLocation()` ?

Reply:
Done